src/utils/handlers/general/searchTrack.ts

```diff
@@ -17,7 +17,7 @@ export async function searchTrack(
 
     const queryData = checkQuery(query);
     if (queryData.isURL) {
-        const url = new URL(query);
+        let url = new URL(query);
```

Comment:
```suggestion
        const url = new URL(query);
```
Use const instead

Reply:
@mzrtamp the use of `let` in his solution is correct since he changed the value of `url` variable on line 81.

Reply:
but, the declaration of `url` can be changed to `const` after my change request is done.